release-notes/next.yml

```diff
@@ -7,10 +7,10 @@ releases:
     issues:
       - text: "container image reference incorrect in `next` `42.20250331.1.0`"
         url: "https://github.com/coreos/fedora-coreos-tracker/issues/1913"
+  42.20250331.1.0:
+    issues:
```

Comment:
According to https://github.com/coreos/fedora-coreos-streams/commit/7123f6583bc95218a5973895be849686d7d4a06b this wasn't added to the stream metadata until `42.20250407.1.0`.